theano/gpuarray/dnn.py

```diff
@@ -354,7 +354,9 @@ class GpuDnnConvDesc(COp):
     See the doc of :func:`dnn_conv` for a description of the parameters
 
     """
-
+    
+    prec_GpuDnn = get_precision(precision, [img, kerns])
```

Comment:
I don't know how this commit can work. Where img and kerns come from?

Anyway, I fixed Friday: https://github.com/Theano/Theano/pull/5650 and the fix is merged.

thanks for trying to fix this. I wanted to have this fixed quickly for the rc3.